src/ansible_builder/containerfile.py

```diff
@@ -434,6 +434,10 @@ def _prepare_galaxy_install_steps(self) -> None:
             # in the final image.
             env = "ANSIBLE_GALAXY_DISABLE_GPG_VERIFY=1 "
 
+        # If nothing actually gets installed, make sure this directory will exist
+        # to prevent the COPY step from failing later.
+        self.steps.append(f"RUN mkdir {os.path.dirname(constants.base_collections_path.rstrip('/'))}")
```

Comment:
Should this include `-p` also, in case the base image might already have this path?

Reply:
fo' sho'